### PR TITLE
open external links in new tab

### PR DESCRIPTION
**Pull request by k35o**
resolved https://github.com/jser/jser.info/issues/121

### CODE REVIEW

File: public/js/external-links-new-tab.js
Comment (k35o) [inline public/js/external-links-new-tab.js:1]: The implementation was based on [switch-alt-link](https://github.com/jser/jser.github.io/blob/develop/public/js/switch-alt-link.js) and https://github.com/jser/jser.github.io/commit/8d3c6697ec321f182f78b0f67857b03532bf913f.

**Review by azu — APPROVED** [review]
LGTM

**Comment by azu** [comment]
ありがとうございます！

**Comment by k35o** [comment]
ちゃんと動いてそうでした🎉

**Pull request merged**